bpf/kmesh/ads/include/route_config.h

```diff
@@ -377,13 +377,13 @@ int route_config_manager(ctx_buff_t *ctx)
 
     virt_host = virtual_host_match(route_config, &addr, ctx);
     if (!virt_host) {
-        BPF_LOG(ERR, ROUTER_CONFIG, "failed to match virtual host, addr=%u\n", addr.ipv4);
+        BPF_LOG(ERR, ROUTER_CONFIG, "failed to match virtual host, addr=%pI4h\n", &addr.ipv4);
```

Comment:
Is it just easy to find the faulty object with address? Does need another identity?

Reply:
Maybe it could find which clusterIP service is failed when do virt host match. But this pr is just to change the ipaddr format of bpf log.

Reply:
ok